pennylane/compiler/qjit_api.py

```diff
@@ -111,3 +112,173 @@ def circuit(theta):
     compilers = AvailableCompilers.names_entrypoints
     qjit_loader = compilers[compiler]["qjit"].load()
     return qjit_loader(fn=fn, *args, **kwargs)
+
+
+def while_loop(*args, **kwargs):
```

Comment:
Why do we go for `*args, **kwargs` here and not the actual arguments?

Reply:
I just used these generic arguments to make these wrappers compatible with all future changes in the signature of `catalyst.while_loop`.

Reply:
Fixed [c2df222](https://github.com/PennyLaneAI/pennylane/pull/4698/commits/c2df22226b69f4397f1546e4d4115344875c834a)